CHARTER.md

```diff
@@ -1,107 +1,121 @@
+- g. The current TSC Chair is:
 
-- f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.  **The TSC Chair, or any other TSC member so designated by the TSC, will serve as the Technical Initiative’s voting representative on the OpenSSF’s Technical Advisory Council (the "TAC").
+  - Eddie Knight, Sonatype (@eddie-knight)
 
-- g. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Technical Initiative, which may include:
+## 3. Technical Initiatives
 
-   - i. coordinating the direction of the Technical Initiative;
+- a. A Technical Initiative is a project or special interest group (SIG) that falls under the oversight of this working group at creation or was subsequently added to this section of the charter.
+  
+  - No other work streams, activities, or bodies of any kind are considered Technical Initiatives, regardless of their affiliation with the WG.
 
-   - ii. approving, organizing or removing activities and projects;
+- b. Technical Initiatives must have a clearly identified lead who, for voting purposes, is not a lead on another ORBIT WG Technical Initiative.
 
-   - iii. establish community norms, workflows, processes, release requirements, and templates for the operation of the Technical Initiative;
+  - i. Leads will report to and participate in the routine activities of the TSC.
+  - ii. Leads may nominate their own replacement at any time, subject to the charter of the Technical Initiative.
+  - iii. In the event that the TSC elects to remove an acting lead, it must appoint a replacement in the same election.
 
-   - iv. establish a fundraising model, and approve or modify a Technical Initiative budget, subject to OpenSSF Governing Board approval;
+- c. A new Technical Initiative should only be brought into the WG after it is confirmed to be topically relevant, and interoperable with no less than two other Technical Initiatives in the WG. Acceptance must be confirmed on the proposal with a 2/3 majority approval from the Technical Steering Committee. That proposal must include an update to this document.
 
-   - v. appointing representatives to work with other open source or open standards communities;
+- d. A Technical Initiative may be de-scoped or archived for any reason following confirmation on the proposal with a 2/3 majority approval from the Technical Steering Committee. That proposal must include an update to this document.
 
-   - vi. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
+- e. The current Technical Initiatives in scope for this WG are:
```

Comment:
Minor suggestion: add links to the Technical Initiatives listed below

Reply:
agreed

Reply:
Should this also include the leads for each TI. In my reading of the above, these folks compose the TSC. Is that correct?

Reply:
The TSC is composed of the leads, yes. But the intent in omitting that reference is to allow the TI freedom to organize itself without going through WG approval for governance decisions

Reply:
fair enough!